jiant/preprocess.py

```diff
@@ -552,20 +572,27 @@ def add_task_label_vocab(vocab, task):
         vocab.add_token_to_namespace(label, namespace)
 
 
-def add_bert_wpm_vocab(vocab, bert_model_name):
-    """Add BERT WPM vocabulary for use with pre-tokenized data.
+def add_pytorch_transformers_wpm_vocab(vocab, tokenizer_name):
+    """Add BERT/XLNet WPM vocabulary for use with pre-tokenized data.
 
-    BertTokenizer has a convert_tokens_to_ids method, but this doesn't do
-    anything special so we can just use the standard indexers.
+    These tokenizers have a convert_tokens_to_ids method, but this doesn't do
+    anything special, so we can just use the standard indexers.
     """
-    from pytorch_pretrained_bert import BertTokenizer
+    do_lower_case = "uncased" in tokenizer_name
+
+    if tokenizer_name.startswith("bert"):
+        from pytorch_transformers import BertTokenizer
+
+        tokenizer = BertTokenizer.from_pretrained(tokenizer_name, do_lower_case=do_lower_case)
```

Comment:
Nit: isn't each BERT version already specifying the casing ?

Reply:
It is, but BertTokenizer requires us to specify it directly. If we don't, it'll give a warning.